pom.xml

```diff
@@ -36,7 +36,7 @@
 
         <maven.build.sourceVersion>11</maven.build.sourceVersion>
 
-        <dependency.alfresco-data-model.version>8.103</dependency.alfresco-data-model.version>
+        <dependency.alfresco-data-model.version>8.104</dependency.alfresco-data-model.version>
```

Comment:
**8.105** is now the latest _alfresco-data-model_ release.

Reply:
Thanks @CezarLeahu